packages/patternfly-4/react-core/src/components/Modal/ModalBox.js

```diff
@@ -26,12 +26,12 @@ const defaultProps = {
   isSmall: false
 };
 
-const ModalBox = ({ children, className, isLarge, isSmall, title, id, ...props }) => (
+const ModalBox = ({ children, className, isLarge, isSmall, title, ariaDescribedById, ...props }) => (
   <div
     {...props}
```

Comment:
can you spread the props at the end to allow for overwriting attribute values.

Reply:
@tlabaj just seeing this comment. Are they not already being spread at the end? I'm not sure what you mean

Reply:
you can ignore that since the id is still coming in explicitly.